config/default/kustomization.yaml

```diff
@@ -12,7 +12,8 @@ namespace: spanner-autoscaler
 #commonLabels:
 #  someName: someValue
 
-bases:
+resources:
+- Kptfile
```

Comment:
Thanks for trying it out.

I have now fixed it by moving the `Kptfile` to a new directory so that it does not interfere with the existing Kustomize resources. I have also added a note for using `kubectl` along with `kpt`. Please have a look at the PR again.